Makefile

```diff
@@ -1,8 +1,21 @@
 default: lintf
 
+DB_NAME ?= runtime
+DB_USER ?= `whoami`
+DB_HOSTNAME ?= localhost
+RD_NAME ?= runtime
+RD_USER ?= `whoami`
+RD_HOSTNAME ?= localhost
+DB_URL="postgresql://$(DB_USER)@$(DB_HOSTNAME):5432/$(DB_NAME)"
+RD_URL="redis://$(RD_USER)@$(RD_HOSTNAME):6379/$(RD_NAME)?pool=$(RD_POOL)&process=$(RD_NUM_PROCESSES)"
```

Comment:
would be better to add prefix "TEST_" on these vars, to avoid confusion

Reply:
ok